metrics/tarantool/luajit.lua

```diff
@@ -0,0 +1,69 @@
+local metrics = require('metrics')
+
+local has_mics_module, misc = pcall(require, 'misc')
```

Comment:
Side note: <misc> is already loaded in the new Tarantool versions.

Reply:
Currently there is no one released version of Tarantool with misc library. This code preserve compatibility with old tarantool versions that aren't support misc module simply skipping this module.

Reply:
I just meant you can simply check that `misc` is not `nil`, nevermind :)

Reply:
But it raises:
```
Tarantool Enterprise For Mac 2.4.2-12-g11f587c91
type 'help' for interactive help
tarantool> require('misc')
---
- error: "module 'misc' not found:No LuaRocks module found for misc\n\tno field package.preload['misc']\n\tno
    file '/Users/o.babin/Projects/tdg/misc.lua'\n\tno file '/Users/o.babin/Projects/tdg/misc/init.lua'\n\tno
```

Reply:
Oh, I've understand you. It's loaded by default globally.
```
Tarantool 2.6.0-172-gc798e18c0
type 'help' for interactive help
tarantool> misc
---
- getmetrics: 'function: builtin#203'
...
```

Reply:
Well, I see we're on different pages. Instead of `require`-ing under `pcall`
```lua
local has_mics_module, misc = pcall(require, 'misc')
```
you can simply do the following:
```lua
local has_misc_module = misc ~= nil
```
If you consider your approach more convenient, feel free to ignore mine.

Reply:
Yes, I got you.

So, it's not so simply. In strict mode or on debug build (where strict mode is enabled by default) with older versions this code will raise:
```lua
tarantool> require('strict').on()
tarantool> misc
---
- error: '[string "return mics"]:1: variable ''misc'' is not declared'
...
```

Finally, I should check `local has_misc_module = rawget(_G, 'misc')`. It's a bit shorter than a variant with require. But yes, it's a question of taste. I think we should check module existence via require.